priv/big_query/migrations/20240612152001_add_zones_unfilled.exs

```diff
@@ -0,0 +1,32 @@
+defmodule BigQuery.Migrations.AddPricing do
+  alias BigQuery.Base.Query
+
+  def up do
+    Query.log("""
+      ALTER TABLE dt_downloads ADD COLUMN zones_filled STRUCT<
+        pre INT64,
+        mid INT64,
+        post INT64,
+        house_pre INT64,
+        house_mid INT64,
+        house_post INT64
+      >;
+
+      ALTER TABLE dt_downloads ADD COLUMN zones_unfilled STRUCT<
+        pre INT64,
+        mid INT64,
+        post INT64,
+        house_pre INT64,
+        house_mid INT64,
+        house_post INT64
+      >;
```

Comment:
So the advantage of structs is: avoid adding 12 columns, and easy to query.

```SQL
SELECT SUM(zones_filled.mid), SUM(zones_unfilled.mid) FROM dt_downloads ...
```

The disadvantage is: they're a bit hard to change later?  We'd probably need to add a new column and shuffle data around if we, say, wanted to add a sonic_id fill rate.  (Do we want to track sonic_id fill rate?).

Also, I did try this from the analytics-ingest-lambda, and you just have to construct an `{pre: 1, mid: 0, ...}` object to get the JSON inserts to work.

Reply:
I've confirmed with sales that we don't need/want any sort of sonic id fill rate.  Just house/paid and pre/mid/post.

Given that, I see no real functional difference between 12 columns and 2 structs.  (Selecting structs in BQ doesn't require any additional syntax ... just `field.struct_key`).  So I really have no strong opinion on which.